elastic/kibana
# [Obs UX] Include with_filters and with_query to hosts count event

closes https://github.com/elastic/kibana/issues/173077

## Summary

Adds `with_query` and `with_filters` attributes to the "Host View Total Host Count Retrieved" event. 

**note**: date picker isn't considered for the flags above.

### How to test

1. Start  ES, Kibana, and  metricbeat
2. Navigate to Infrastructure > Hosts
3. Inspect the `kibana-browser` request payload and check if the new attributes are there
4. Add/remove filters and query and repeat step 3.
